src/Symfony/Component/Uid/Uuid.php

```diff
@@ -182,4 +184,36 @@ private static function format(string $uuid, string $version): string
 
         return substr_replace($uuid, '-', 23, 0);
     }
+
+    /**
+     * Transforms a binary string, a base-32 string or a base-58 string to a RFC4122 string.
+     *
+     * @param int-mask-of<Uuid::FORMAT_*> $format
+     *
+     * @return non-empty-string
```

Comment:
@alexandre-daubois this is currently not true, afaik ... should we enforce it (or remove it) ?

Reply:
Indeed, but more than that, it's a bit confusing. I proposed this update: https://github.com/symfony/symfony/pull/58089